metrics/web/lib/base/presentation/widgets/dropdown_item.dart

```diff
@@ -30,49 +32,38 @@ class DropdownItem extends StatefulWidget {
 
   /// Creates a widget that displays a dropdown item of the [DropdownMenu].
   ///
-  /// The [child] must not be `null`.
+  /// The [builder] must not be `null`.
   const DropdownItem({
     Key key,
-    @required this.child,
+    @required this.builder,
     this.width,
     this.height,
     this.alignment,
     this.padding,
     this.backgroundColor,
     this.hoverColor,
-  })  : assert(child != null),
+  })  : assert(builder != null),
         super(key: key);
 
   @override
   _DropdownItemState createState() => _DropdownItemState();
 }
 
 class _DropdownItemState extends State<DropdownItem> {
-  /// Indicates whether this widget is hovered or not.
-  bool _isHovered = false;
-
   @override
   Widget build(BuildContext context) {
-    final color = _isHovered ? widget.hoverColor : widget.backgroundColor;
-
-    return HandCursor(
-      child: MouseRegion(
-        onEnter: (_) => _changeHover(true),
-        onExit: (_) => _changeHover(false),
-        child: Container(
+    return TappableArea(
+      mouseCursor: MouseCursor.click,
+      builder: (context, bool isHovered) {
```

Comment:
```suggestion
      builder: (context, isHovered) {
```
We can omit type declaration here.